pseudo.txt

```diff
@@ -0,0 +1,7 @@
+GET all icecream flavors "/icecream/flavors"
```

Comment:
"/icecreams/flavors" would return all of the flavors in your DB. "/icecreams" would more likely return all of the ice cream objects.